tests/search/xgboost/xgboost.rb

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'indexed_search_test'
+require 'json'
+
+class XGBoostServing < IndexedSearchTest
+
+  def setup
+    @valgrid = false
+    set_owner("jobergum")
+    set_description("Test XGBoost model representation in Vespa")
+
+    @node = @vespa.nodeproxies.values.first
+    @node.execute("pip3 install xgboost sklearn")
```

Comment:
It would be best to have these as part of our systemtest image instead of installing stuff in the test itself. But, I guess it's ok for now.

Reply:
I chose this approach to test with whatever is the newest stable release in case of changes.